### PR TITLE
Move admin action log type list generation to helper

### PR DESCRIPTION
**Pull request by mjankowski**
This PR is just the method extraction, but thoughts on some/any of:

- Sort this list alphabetically by target_type first, then by action?
- Sort that way, but then also do a group by (model type) and used a grouped select in the view?

The long/large list here is somewhat hard to parse in the dropdown.

### CODE REVIEW

**Comment by oneiros** [comment]
I have never actually used this feature, but I believe simply sorting alphabetically might already help a lot.

**Comment by oneiros** [comment]
Would you like add sorting to this PR, @mjankowski?

**Comment by oneiros** [comment]
> Sure -- just sort by the option strings? By the target type? Maybe type first and then action within the type (but don't yet group?)

Again, I think just sorting by the option strings would be an improvement. I cannot comment on the other ideas. They sound sensible, but I really think someone should comment on this who has actually used this dropdown before :slightly_smiling_face:

**Comment by mjankowski** [comment]
Added a plain sort on the i18n'd option values:

Screenshot 2024-10-02 at 10 33 30

May try a group by as followup if it looks better to me.

**Review by oneiros — APPROVED** [review]
